src/server/server.js

```diff
@@ -394,4 +404,58 @@ function processPluginHtml(html, pluginId) {
     });
 }
 
+function localizeHtmlFile(htmlFile, html, lang) {
+    // Replace localized strings
+    var targetJsonFile = getI18nJsonFile(htmlFile, lang);
+    if (utils.existsSync(targetJsonFile)) {
+        try {
+            var locItems = require(targetJsonFile);
+        } catch (e) {
+            // Don't care if it fails
+            return;
+        }
+
+        locItems.forEach(function (locItem) {
+            html = replaceLocalizationItem(html, locItem);
+        });
+    }
+    return html;
+}
+
+function replaceLocalizationItem(html, locItem) {
+    // To try and be as targeted as possible, we replace attribute by looking for 'attribute="value"', and
```

Comment:
Yeah, my thoughts on this were as follows:
* My main focus here was getting the translation cycle in place (around the xliff files) ASAP so the loc team could begin their work, rather than necessarily getting the replacement process perfect.
* It works with what we have now.
* Any HTML files that contain strings to be translated are actually rewritten by the `update` process (to insert the loc id attribute), which will clean up white space and quotes.
* If I was going to stick with this approach, I would use a regex here, and/or add some HTML linting to our test process (though the linting that wouldn't help UI defined in plugins).
* If I move to the "generated localized files in the update process" approach, this problem goes away.

Bottom line is my preference would be to leave this as is for now, and iterate on it once we have kicked off the translation process.